src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -2086,10 +2079,11 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
             //
 
             // Adding the value of $(MSBuildExtensionsPath*) property to the list of search paths
-            var prop = _data.GetProperty(refKindInProject.StringRepresentation);
+            var prop = _data.GetProperty(fallbackSearchPathMatch.PropertyName);
+            var fallbackExtensionPaths = fallbackSearchPathMatch.SearchPaths.ToList();
```

Comment:
should this `ToList` be moved to ProjectImportPathMatch construction time so you don't have to do it for every import?